app/containers/codeArea/index.js

```diff
@@ -42,7 +42,7 @@ export default class CodeArea extends Component {
       highlighted. The following code fixes it by appending <span> to each line
       of the comment block.
     */
-    const commentPattern = /<span class='hljs-comment'>(.|\n)*?<\/span>/g
+    const commentPattern = /<span class="hljs-comment">(.|\n)*?<\/span>/g
```

Comment:
Cool! What changing to double quotes would work here?

Reply:
In current version, we have a block:

https://github.com/hackjutsu/Lepton/blob/caa29ebd9d333a76b79bdb7409978b82511c539d/app/containers/codeArea/index.js#L38:L51

As comment stated, Highlight.js wraps comment blocks inside span with class `hljs-comment` but the problem is that generated html by Highlightjs uses double quote instead of single quote. (`<span class="hljs-comment">`). Current version tries to replace `highlightedContent` with single quoted `commentPattern` and it does not match anything actually and not trigger the _inner replace callback_ function. So, using double quoted `commentPattern` forces to change each line of multiline comment block.

Reply:
By the way, my PR solves the simple multiline block comment except lines include span with `hljs-doctag` class. Working on that...

<img width="785" alt="image" src="https://user-images.githubusercontent.com/965804/45600733-2d76cd80-ba0a-11e8-915d-48f561939aaa.png">

Reply:
Can you check this again? Now, it also checks `*/` characters with closing span in `comment Pattern`.